bundles/org.openhab.binding.webexteams/DEVELOP.md

```diff
@@ -0,0 +1,35 @@
+# Development overview
```

Comment:
This file is helpful for development of a single binding. But should not be in this repository for all bindings as it contains generic information available in other places. Better to remove this file from this pr.

Reply:
Done.

Reply:
This is not yet done, the file is still there.

Reply:
Made another attempt to delete.  I *think* it's gone now.